apps/meteor/ee/server/models/raw/LivechatPriority.ts

```diff
@@ -48,12 +48,16 @@ export class LivechatPriorityRaw extends BaseRaw<ILivechatPriority> implements I
 		return this.findOne(query, options);
 	}
 
+	findByDirty(): FindCursor<Pick<ILivechatPriority, '_id'>> {
```

Comment:
This has a projection by default so it's not returning a ILivechatPriority but a Pick<ILivechatPriority, _id>

Reply:
edit: i'm dum